pyproject.toml

```diff
@@ -195,6 +195,7 @@ filterwarnings = [
     # Ignore a warning we emit about not supporting the parallel",
     # reading option for now, can be removed once the issue is fixed",
     "ignore:parallel reading does not currently work, so falling back to serial",
+    "ignore:assertions not in test modules or plugins will be ignored"
```

Comment:
This needs a comment that would explain under which conditions this warning can occur.